src/Okta.Sdk.IntegrationTests/PoliciesScenarios.cs

```diff
@@ -432,6 +462,156 @@ public async Task CreatePasswordPolicyRule()
             }
         }
 
+        [Fact]
+        public async Task CreateAccessPolicyPolicyRule()
+        {
+            var client = TestClient.Create();
+            var guid = Guid.NewGuid();
+
+            var createdApp = await client.Applications.CreateApplicationAsync(new CreateOpenIdConnectApplication
+            {
+                Label = $"dotnet-sdk: AddOpenIdConnectApp {guid}",
+                ClientUri = "https://example.com/client",
+                LogoUri = "https://example.com/assets/images/logo-new.png",
+                ResponseTypes = new List<OAuthResponseType>
+                {
+                    OAuthResponseType.Token,
+                    OAuthResponseType.IdToken,
+                    OAuthResponseType.Code,
+                },
+                RedirectUris = new List<string>
+                {
+                    "https://example.com/oauth2/callback",
+                    "myapp://callback",
+                },
+                PostLogoutRedirectUris = new List<string>
+                {
+                    "https://example.com/postlogout",
+                    "myapp://postlogoutcallback",
+                },
+                GrantTypes = new List<OAuthGrantType>
+                {
+                    OAuthGrantType.Implicit,
+                    OAuthGrantType.AuthorizationCode,
+                },
+                ApplicationType = OpenIdConnectApplicationType.Web,
+            });
+
+            var accessPolicyId = createdApp.GetAccessPolicyId();
+
```

Comment:
Should we assert/verify for non null accessPolicyId here? If null, do not proceed with rest of the test?

Reply:
There's always an access policy by default when you create an application.